platform/ios/app/MBXViewController.m

```diff
@@ -1551,6 +1569,71 @@ - (void)testQueryPointAnnotations {
     [self presentViewController:alertController animated:YES completion:nil];
 }
 
+- (id<MGLAnnotation>)randomOffscreenPointAnnotation {
+
+    NSPredicate *pointAnnotationPredicate = [NSPredicate predicateWithBlock:^BOOL(id  _Nullable evaluatedObject, NSDictionary<NSString *,id> * _Nullable bindings) {
+        return [evaluatedObject isKindOfClass:[MGLPointAnnotation class]];
+    }];
+
+    NSArray *annotations = [self.mapView.annotations filteredArrayUsingPredicate:pointAnnotationPredicate];
+
+    if (annotations.count == 0)
```

Comment:
Nit: The return statement should be at the end of this line or between curly brackets.
`if (annotations.count == 0) return nil;`